README.md

```diff
@@ -148,8 +148,8 @@ Right now, to my knowledge the Ferrocene Spec does not go into a lot of detail o
 
 ### What about Miri?
 
-MiniRust is in, at least conceptually, very similar to [Miri](https://github.com/rust-lang/miri).
-You can think of it as "idealized Miri": if Miri didn't have to care about working with all the rustc data structures that represent MIR and types, and didn't care about performance nor diagnostics, then it would be implemented like this specification.
+MiniRust is, at least conceptually, very similar to [Miri](https://github.com/rust-lang/miri).
+You can think of it as an "idealized Miri": if Miri didn't have to care about working with all the rustc data structures that represent MIR and types, and cared about neither performance nor diagnostics, then it would be implemented like this specification.
```

Comment:
Is the "an" here required"? To me, the sentence "MiniRust is idealized Miri" sounds better than "MiniRust is an idealized Miri" -- an indefinite article sounds somewhat odd here IMO, "being a Miri" is not a thing.